games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,115 @@
+SUMMARY="An Amazing 3D Flight Simulator"
+DESCRIPTION="FlightGear creates a sophisticated flight simulator framework \
+for use in research or academic environments, pilot training, as an industry \
+engineering tool, for DIY-ers to pursue their favorite interesting flight \
+simulation idea, and last but certainly not least as a fun, realistic, and \
+challenging desktop flight simulator. We are developing a sophisticated, open \
+simulation framework that can be expanded and improved upon by anyone interested \
+in contributing."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://mirrors.ibiblio.org/flightgear/ftp/release-2018.2/flightgear-2018.2.2.tar.bz2"
```

Comment:
Typo fix: missing "t".
```diff
-SOURCE_URI="http://mirrors.ibiblio.org/flightgear/ftp/release-2018.2/flightgear-2018.2.2.tar.bz2"
-SOURCE_URI="https://downloads.sourceforge.net/flightgear/flighgear-$portVersion.tar.bz2"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flightgear-$portVersion.tar.bz2"
```
(SourceForge does not have fli**gh**gear but it has fli**ght**gear :)